cmd/fluxctl/install_cmd.go

```diff
@@ -51,6 +51,7 @@ fluxctl install --git-url 'git@github.com:<your username>/flux-get-started' | ku
 	cmd.Flags().BoolVar(&opts.RegistryScanning, "registry-scanning", true,
 		"scan container image registries to fill in the registry cache")
 	cmd.Flags().StringVarP(&opts.outputDir, "output-dir", "o", "", "a directory in which to write individual manifests, rather than printing to stdout")
+	cmd.Flags().BoolVar(&opts.NoSecurityContext, "no-security-context", false, "Ensure that no security context information is added to the pod specs. Defaults to 'false'")
```

Comment:
I think negatives are confusing. How about `--add-security-context` and defaulting it to `true`?

Reply:
Yes it is confusing, very good point. I have changed this.